.github/workflows/documentation.yml

```
@@ -0,0 +1,26 @@
+name: Documentation
+
+on:
+  push:
+    branches:
+      - master
+    tags: '*'
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: julia-actions/setup-julia@v1
+        with:
+          version: 1
+      - name: Install dependencies
+        run: julia --project=docs/ -e '
+              using Pkg; 
+              Pkg.develop(PackageSpec(path=pwd())); 
+              Pkg.instantiate()'
+      - name: Build and deploy
+        env:
```

Not sure if it has to be specified explicitly, but it's mentioned in the Documenter docs:
```suggestion
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```